proposals/specification/interop.jsonld

```diff
@@ -86,6 +74,133 @@
       "@id": "interop:hasAccessGrant",
       "@type": "@id"
     },
+    "grantedBy": {
+      "@id": "interop:grantedBy",
+      "@type": "@id"
+    },
+    "grantedAt": {
+      "@id": "interop:grantedAt",
+      "@type": "xsd:dateTime"
+    },
+    "grantee": {
+      "@id": "interop:grantee",
+      "@type": "@id"
+    },
+    "hasAccessNeedGroup": {
+      "@id": "interop:hasAccessNeedGroup",
+      "@type": "@id",
+      "@container": [
+        "@id",
+        "@set"
+      ]
+    },
```

Comment:
I think we should only keep one otherwise I'm not sure which one gets picked up during compacting.
To be honest I think we should just dish the camel case ones and accept mixed casing patterns in a single document.

Reply:
Actually, I meant to say that we should dish the snake case version 🐍. Should have imagine the animal before posting 🐫 .

Gland no kebab cases involved so far 🍢 😅